pcftests/pcfcfg.yaml

```
@@ -4,6 +4,7 @@
 configuration:
   defaultBdtRefId: BdtPolicyId-
   nrfUri: https://nrf:443
+  enableNrfCaching: true
```

Should we add the variable in the yaml file and set a default value? Should it be `900 (15mins)` as it done in the `init_context.go`?
```suggestion
  enableNrfCaching: true
  nrfCacheEvictionInterval: <some-value>
```

---

Default value is already set in the init_context.go, hence we did not add  a variable here. However, we could do it.

---

Done